src/Kusto/README.md

```diff
@@ -53,13 +53,13 @@ require:
   - $(this-folder)/../readme.azure.noprofile.md
 # lock the commit
 input-file:
-  - $(repo)/specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2021-08-27/kusto.json
-branch: 2995d5c7da8f7626449bff01d8a7d306df1e6e6d
+  - C:\GitHub\azure-rest-api-specs\specification\azure-kusto\resource-manager\Microsoft.Kusto\stable\2022-02-01\kusto.json
```

Comment:
Please follow the original way to specify the swagger file and its commit ID - we do not allow generating basing on a local file